api/Navigator.json

```diff
@@ -1112,15 +1112,29 @@
       "getInstalledRelatedApps": {
         "__compat": {
           "support": {
-            "chrome": {
-              "version_added": "80"
-            },
+            "chrome": [
+              {
+                "version_added": "85",
+                "notes": "Only supported on Windows. Resolves with an empty array on other platforms."
```

Comment:
I'm not well-versed in this API but I think this might be a case where partial_implementation applies: https://github.com/mdn/browser-compat-data/blob/master/docs/data-guidelines.md#operating-system-limitations-imply-partial_implementation I'd be interested in getting your take on this

Reply:
I also don't know this API well, but I think it could be argued both ways. A native app has to be associated with an origin in some way to be related, and I guess that's an app store feature or some metadata in the installed app. Both depend on some changes to the OS I presume. So there are probably some platforms where there can't be any related apps by definition, and then returning an empty array is correct, not an incomplete implementation.

However, I don't know what the state of this on all platforms are, so I can't say if implementation is possible or not.

Calling it partial seems OK with me, it calls attention to the problem and isn't clearly wrong.

Reply:
I've added `partial_implementation`, and also `version_removed` even though nothing was really removed, but the initial note applies only to a range of version.